### PR TITLE
Using a flask blueprint for configuring the health and metrics views/…

### PR DESCRIPTION
**Pull request by dehort**
…routes

Removed the /health endpoint from the api swagger doc

### CODE REVIEW

**Comment by dehort** [comment]
This PR removes the health check from the api.  To me, the health endpoint and metrics endpoint are more of an artifact of where the inventory service will be deployed than part of the actual api.

This PR moves the health and metrics endpoints to the following urls:

/r/insights/platform/inventory/health
/r/insights/platform/inventory/metrics

File: README.md [hunk 1]
```diff
@@ -46,12 +46,13 @@ Running the tests is quite simple:
...
 
 ```
-gunicorn --log-level=debug run
+gunicorn --log-level=debug -c gunicorn.conf run
 ```
 
 Configuration system properties:
 
 ```
+ prometheus_multiproc_dir=/path/to/prometheus_dir
```
[the remark this answers is not in the document]
Comment (dehort) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -46,12 +46,13 @@ Running the tests is quite simple:
 Running the server:
 
 ```
-gunicorn --log-level=debug run
+gunicorn --log-level=debug -c gunicorn.conf run
```
[the remark this answers is not in the document]
Comment (dehort) [reply README.md]: Done.

**Comment by Glutexo** [comment]
Thanks for the changes! 👍

**Comment by Glutexo** [comment]
I created a little pull request #66 to make the running of the local development easier. It utilizes Python’s [_TemporaryDirectory_](https://docs.python.org/3.7/library/tempfile.html#tempfile.TemporaryDirectory) to create and cleanup the folder Prometheus needs.

**Comment by dehort** [comment]
Merged.

**Pull request closed without merging**